overlord/snapstate/snapstate.go

```diff
@@ -452,14 +452,30 @@ func defaultContentPlugProviders(st *state.State, info *snap.Info) []string {
 	return out
 }
 
+func getBoolFeatureFlag(tr *config.Transaction, flag string) (bool, error) {
+	var v interface{} = false
```

Comment:
Is the initialization here actually needed? I mean, it looks like we always return an explicit false unless we actually have a value?

Reply:
It just avoids the extra `if == nil { return false, nil }` or a `case nil: ..` handling. No strong opinion here, can add it.